injector/cpuset_manager.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package injector
+
+import (
+	"fmt"
+	"github.com/DataDog/chaos-controller/cpuset"
+	"sync"
+)
+
+type StresserManager interface {
+	TrackInjectorCores(config CPUPressureInjectorConfig) error
+	GetCoresToBeStressed() cpuset.CPUSet
+	IsCoreAlreadyStressed(core int) bool
+	TrackStresser(core int, stresserPID int)
+	GetStresserPIDs() map[int]int
+}
+
+type cpuStressserManager struct {
+	mutex              sync.Mutex
+	coresToBeStressed  cpuset.CPUSet
+	stresserPIDPerCore map[int]int
+}
+
+func NewCPUStresserManager() StresserManager {
+	return &cpuStressserManager{
+		coresToBeStressed:  cpuset.NewCPUSet(),
+		stresserPIDPerCore: map[int]int{},
+	}
+}
+
+func (manager *cpuStressserManager) setCoresToBeStressed(cores cpuset.CPUSet) {
+	manager.mutex.Lock()
+	defer manager.mutex.Unlock()
+	manager.coresToBeStressed = cores
+}
+
+func (manager *cpuStressserManager) GetCoresToBeStressed() cpuset.CPUSet {
+	return manager.coresToBeStressed
+}
+
+func (manager *cpuStressserManager) IsCoreAlreadyStressed(core int) bool {
+	_, isCoreStressed := manager.stresserPIDPerCore[core]
+	return isCoreStressed
+}
+
+func (manager *cpuStressserManager) TrackStresser(core int, stresserPID int) {
```

Comment:
As most of other methods are named another way, I would change the name to something like:
```suggestion
func (manager *cpuStressserManager) TrackCoreAlreadyStressed(core int, stresserPID int) {
```